docs/breadcrumb/toc.yml

```diff
@@ -25,6 +25,9 @@
       - name: JavaScript
         tocHref: /onedrive/developer/controls/file-pickers/js-v72
         topicHref: /onedrive/developer/controls/file-pickers/js-v72/index
+      - name: JavaScript (Preview)
+        tocHref: /onedrive/developer/controls/file-pickers/npm
```

Comment:
maybe we should call it react since it's literally a react component? i'm still feeling weird about categorizing it as 'npm' (which is a distribution mechanism). maybe 'react' or maybe some other term?

Reply:
Fantastic idea!